source/js/foundation/pages/mozfest/tito.js

```diff
@@ -1,17 +1,20 @@
 /**
- * Using the tito library to set up a ticket purchasing widget.
```

Comment:
This file didn't seem to be used/imported anywhere in the codebase. I'm reviving it.